tests/python/gaia-ui-tests/gaiatest/apps/camera/app.py

```diff
@@ -87,7 +87,7 @@ def tap_select_button(self):
         self.tap_element_from_system_app(select)
 
         # Fall back to app beneath the picker
-        Wait(self.marionette).until(lambda m: self.apps.displayed_app.name != self.name)
+        Wait(self.marionette).until(lambda m: self.apps.displayed_app.manifest_url != self.manifest_url)
```

Comment:
Oh, sorry, this is a not equals test, you can't use wait_to_be_displayed for that.